Adds -fcommon to CFLAGS to allow compilation on more recent GCC.
diff --git a/system/Make.sys-shared b/system/Make.sys-shared
index 51891e7..f89f876 100644
--- a/system/Make.sys-shared
+++ b/system/Make.sys-shared
@@ -19,7 +19,7 @@ MAKE = make
 # C-preprocessor flags
 CPPFLAGS ?=
 # C-compiler flags
-CFLAGS += -ffast-math -funroll-loops -fPIC -pedantic -Wall
+CFLAGS += -ffast-math -funroll-loops -fPIC -pedantic -Wall -fcommon
 
 CC     = gcc 
 #LDLIB  = -ldl
